javascript/jquery/security/audit/prohibit-jquery-html.yaml

```diff
@@ -6,11 +6,16 @@ rules:
       - pattern-not: |
           $X.html("...",...)
     message: >-
-      JQuery's `.html()` is susceptible to Cross Site Scripting (XSS) attacks. 
```

Comment:
Thanks! My proposal is to keep **message** verbose for those who do not use Github alerts and will not see extra information from fileds like `short description` and `help`. WIll this work for you?

Reply:
That does! I will update shortly.